### PR TITLE
COMMUNITY-69 & COMMUNITY-70 | add network isolation for tests containers and fix doc generation

### PR DESCRIPTION
**Pull request by ManuelJNunez**
To fix COMMUNITY-69 I added `--network none` to the docker run arguments to avoid interferences among the tests.
To fix COMMUNITY-70 I created a new directory (`/opt/node_deps`) inside the container and set it as the npm prefix.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,9 +16,11 @@ RUN apt-get update \
     && apt-get install -y gcc g++ git make python3 python3-pip python3-venv \
     && useradd -u 789 -m jenkins
 
-RUN npm install -g npm jsdoc
+RUN mkdir -p /opt/node_deps \
```
Comment (Alxe) [inline Dockerfile:19]: Would it make sense to move `/opt/node_deps/` into an argument? That way we can avoid hardcoding it in the `Jenkinsfile`.

If you change it, consider using a default value for it, e.g. `ARG NODE_PREFIX=/opt/node_deps'`.

**Review by Alxe — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (ManuelJNunez) [reply Dockerfile:19]: It will be hardcoded anyway...

Comment (Alxe) [reply Dockerfile:19]: In the `Jenkinsfile` we could reference it as a variable for the Dockerfile and the Jenkins, e.g.:

```
# Dockerfile
ARG NODE_PREFIX='/opt/note_deps'
...
ENV PATH="...${NODE_PREFIX}.."

# Jenkinsfile
def node_prefix = '/opt/my_personal_deps'
...
additionalBuildArgs  '--build-arg NODE_VERSION=18 NODE_PREFIX=${node_prefix}'
...
sh 'npm config set prefix ${node_prefix}`
```

However, I don't think this is a huge deal, and we can merge with this.

Comment (ManuelJNunez) [reply Dockerfile:19]: I see it hardcoded still.

**Pull request merged**